index.js

```diff
@@ -355,15 +355,11 @@ function Physics (mcData, world) {
       if (isOnLadder(world, pos)) {
         vel.x = math.clamp(-physics.ladderMaxSpeed, vel.x, physics.ladderMaxSpeed)
         vel.z = math.clamp(-physics.ladderMaxSpeed, vel.z, physics.ladderMaxSpeed)
-        vel.y = Math.max(vel.y, entity.control.sneak ? 0 : -physics.ladderMaxSpeed)
+        vel.y = Math.max(entity.control.jump ? physics.ladderClimbSpeed : vel.y, entity.control.sneak ? 0 : -physics.ladderMaxSpeed)
```

Comment:
Pressing jump on a ladder doesn't make you climb. I think the collision check that was there before was correct. It was based on line 214 of https://www.mcpk.wiki/wiki/SourceCode:EntityLivingBase

Reply:
How else is climbing supposed to work?

Reply:
Hum it seems like this feature was introduced in 1.14+ https://www.mcpk.wiki/wiki/Ladders_and_Vines
So indeed, we need to support it but we have to stay compatible with previous versions too.
Back in the days, to climb, it was enough to press forward looking at a ladder or vine.

Reply:
So ill revert the two changes to do with climbing and add a version conditional ability to climb using jump.

Reply:
Sounds good